bcda/cli.go

```diff
@@ -157,3 +148,61 @@ func getCCLFFileMetadata(filePath string) (cclfFileMetadata, error) {
 
 	return metadata, nil
 }
+
+func importCCLFDirectory(filePath string) (success, failure int, err error) {
+	var cclf0, cclf8, cclf9 []cclfFileMetadata
+
+	err = filepath.Walk(filePath, sortCCLFFiles(&cclf0, &cclf8, &cclf9))
+	if err != nil {
+		return 0, 0, err
+	}
+	for _, file := range cclf8 {
+		//todo match cclf0 and validate
+		err = importCCLF8(file)
+		if err != nil {
+			failure += 1
+			continue
+		}
+		log.Info(fmt.Sprintf("Successfully imported CCLF8 file: %v", file))
+		success += 1
+	}
+	for _, file := range cclf9 {
+		//todo match cclf0 and validate
+		err = importCCLF9(file)
+		if err != nil {
+			failure += 1
```

Comment:
Same comment as above... failure logging.

Reply:
I couldn't get the import as it stands to fail.  I imported bad files constantly, but they just roll through.  I suggested a solution in slack, but it seemed kind of out of scope since the imports were already resolved by other pending commits.